exchanges/request-policy/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
```

Comment:
I thought this would have been unnecessary - assuming it breaks if you don't add the mappings?

Reply:
Each package needs this file. I've found that global type checking only works with a `tsconfig` in the monorepo's root. Local type checking for a package requires the local config, but it can inherit from the root one. The paths needs to be defined in both places and need to be different for the typings output to be correct.

This is from memory, so the explanations of why this is this way may not be 100% accurate 😅